play-media-native/screens/MediaListing.tsx

```diff
@@ -0,0 +1,75 @@
+import { useCallback, useMemo } from 'react';
+import { Image, View } from 'react-native';
+import { AnimatedFAB } from 'react-native-paper';
+import { useQuery } from 'react-query';
+
+import { getAllMedia } from '../api/queries/getMedia';
+import { ListingImages } from '../features/ListingImages/ListingImages';
+import { LoadingScreen } from '../features/LoadingScreen/LoadingScreen';
+import { MediaItemCardDisplay } from '../features/MediaItemCardDisplay/MediaItemCardDisplay';
+import { MediaItemListDisplay } from '../features/MediaItemListDisplay/MediaItemListDisplay';
+import { Screen } from '../features/Screen/Screen';
+import { ListingImageDisplayType } from '../features/SelectDisplayButtons/SelectDisplayButtons';
+import { useScrollOffset } from '../hooks/useScrollOffset/useScrollOffset';
+import { Media } from '../interfaces/media';
+import { styles } from '../theme/styles';
+
+const gridItemStyle = {
+  height: 120,
+  flex: 0.5,
+  margin: 2,
+};
+
+export const MediaListingScreen = ({ navigation }) => {
+  const {
+    data: images,
+    isLoading: isFetchingInitialMedia,
+    refetch: refetchMedia,
+    isRefetching: isRefetchingMedia,
+  } = useQuery('media', () => getAllMedia());
```

Comment:
I need to change this after [PR#8511](https://github.com/Sitecore/Sitecore.Demo.CHONE/pull/75) gets merged